src/packages/SpeakClient.ts

```diff
@@ -1,5 +1,5 @@
 import { AbstractRestfulClient } from "./AbstractRestfulClient";
-import { DeepgramError, DeepgramUnknownError, isDeepgramError } from "../lib/errors";
+import { DeepgramError, DeepgramUnknownError } from "../lib/errors";
```

Comment:
A little advice would be to keep your PRs atomic, and change only things relevant to what you want changed.